assets/src/helpers/vehicleLabel.ts

```diff
@@ -1,38 +1,41 @@
-import { isVehicle } from "../models/vehicle"
-import { RunId, VehicleOrGhost } from "../realtime"
+import { isVehicle, isVehicleInScheduledService } from "../models/vehicle"
+import { Ghost, RunId, Vehicle } from "../realtime"
 import {
   UserSettings,
   vehicleLabelSetting,
   VehicleLabelSetting,
 } from "../userSettings"
 
 export const runOrBusNumberLabel = (
-  vehicleOrGhost: VehicleOrGhost,
+  vehicleOrGhost: Vehicle | Ghost,
   settings: UserSettings
 ): string => {
   switch (vehicleLabelSetting(settings, vehicleOrGhost)) {
     case VehicleLabelSetting.RunNumber:
       return runIdToLabel(vehicleOrGhost.runId)
     case VehicleLabelSetting.VehicleNumber:
-      return isVehicle(vehicleOrGhost) ? vehicleOrGhost.label : "N/A"
+      return isVehicle(vehicleOrGhost) ? vehicleOrGhost.label || "N/A" : "N/A"
```

Comment:
suggestion(non-blocking): I think this could be simplified? though I'm not sure this is any better for readability.
```suggestion
      return isVehicle(vehicleOrGhost) && vehicleOrGhost.label || "N/A"
```

Reply:
I dunno if it needs parenthesis to help it understand order though
```suggestion
      return (isVehicle(vehicleOrGhost) && vehicleOrGhost.label || "N/A")
```